src/char/int_party.c

```diff
@@ -614,6 +614,7 @@ int mapif_parse_PartyLeave(int fd, int party_id, int account_id, int char_id)
 {
 	struct party_data *p;
 	int i,j;
+	unsigned int leader;
```

Comment:
the usage of unsigned integers is not preferred unless explicitly needed, please change it to int